Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -45,9 +46,27 @@ const configOptionTitle = (title, description) => {
   );
 };
 
+const timeFrames = [
+  { _id: 1, name: "seconds" },
+  { _id: 2, name: "minutes" },
+  { _id: 3, name: "hours" },
+  { _id: 4, name: "days" },
+];
+
+const sxButtons = {
+  width: 110,
+  height: 34,
+};
+
 const CreateNewMaintenanceWindow = () => {
   const [values, setValues] = useState({
     repeat: 1,
+    date: dayjs(),
+    startTime: dayjs(),
+    duration: "60",
+    unit: "minutes",
```

Comment:
This isn't a valid value for `unit`, so this displays as blank:

![image](https://github.com/user-attachments/assets/57c57f4d-ef04-44ac-b350-ced75c44430f)

The values sepcificed are:
```
 { _id: 1, name: "seconds" },
  { _id: 2, name: "minutes" },
  { _id: 3, name: "hours" },
  { _id: 4, name: "days" },
  ```
  
 So you need to specify one of `1`, `2`, `3`, or `4`.
 
 `minutes` corresponds to `2`, but I think you probably want to start at `seconds` as it is the first choice, so `1` is probably the appropriate value.